ic-canister-client/src/error.rs

```diff
@@ -1,4 +1,5 @@
 use ic_exports::ic_cdk::api::call::RejectionCode;
+use ic_exports::ic_test_state_machine::UserError;
```

Comment:
This brings clippy warning since the import is used under a feature, We can just use full path in the `StatemachineTestError` variant

Reply:
It used in three places, so I added feature condition to the `use ...` itself.